ApplicationLibCode/ReservoirDataModel/RigActiveCellInfo.cpp

```diff
@@ -94,10 +80,10 @@ void RigActiveCellInfo::setCellResultIndex( size_t reservoirCellIndex, size_t re
 
     m_cellIndexToResultIndex[reservoirCellIndex] = reservoirCellResultIndex;
 
-    if ( reservoirCellResultIndex >= m_reservoirCellResultCount )
+    if ( reservoirCellResultIndex >= m_reservoirActiveCellCount )
```

Comment:
I think modification of m_reservoirActiveCellCount should be deleted. Update of this variable is done in setGridActiveCellCounts() and computeDerivedData()